packages/http/README.md

```diff
@@ -237,6 +237,7 @@ const config = {
 ```
 
 With the above configuration the http client will proxy your requests to a server and will validate the response with the OpenAPI you loaded. However, it will not validate the input (e.g. will not check whether the provided query param is valid).
+Please also note, that the proxied requests will have 'user-agent' header set to Prism/<<PRISM_VERSION>>.
```

Comment:
I don't think we need to mention this. I doubt anyone will care, and its very obvious if they do. Most HTTP clients set user agent already, so this is just changing from `User-Agent: axios whatever` to `User-Agent: prism whatever`.

Reply:
Hm.. I don't really see why delete this.

Reply:
I agree with @philsturgeon here — this information does not really bring any value to the consumer; I'd remove it as well.

Reply:
We should do things for a reason, not come up with reasons to not do things. :)

Documentation is important, but keeping a high ratio of signal to noise is important s the more stuff we add to docs which isn’t particularly useful, the more users have to scroll, the higher their chance of missing useful information, which means they’re posting issues, asking for help in support, which is time people could have spent doing other things. 

So let’s delete this. It adds no value. :)

Reply:
okey-dokey :+1: